waku/v2/protocol/waku_noise/noise.nim

```diff
@@ -62,6 +77,94 @@ type
     nonce: ChaChaPolyNonce
     ad: seq[byte]
 
+  #################################
+  # Noise handshake patterns
+  #################################
+
+  # The Noise tokens appearing in Noise (pre)message patterns 
```

Comment:
Do we use the same term in our own spec i.e., pre-message? if not, I'd suggest using consistent terms